dynamics/src/include/cgParametricMomentum.hpp

```diff
@@ -62,6 +64,12 @@ template <int CG> class CGParametricMomentum {
         : smesh(sm)
         , pmap(sm)
     {
+        std::cerr << "CGParametricMomentum can't be used any more. It is replaced by the different "
+                     "Kernels. Reason is that the ParamwetricMomentumMap requires the DGadvection "
+                     "degree as template but this is not provided. It could be easily changed, but "
+                     "instead one should use the Kernel-infrastructure."
```

Comment:
"CGParametricMomentum can't be used any more. It is replaced by the various dynamics kernels. ParametricMomentumMap requires the DGadvection degree as template but this is not available in CGParametricMomentum. Please use the Kernel-infrastructure instead."